libraries/AP_Scripting/applets/Aerobatics/FixedWing/TrajectoryBased/Schedules/README.md

```diff
@@ -0,0 +1,35 @@
+# Scripted Aerobatics Schedules
+
+This directory contains full aerobatic schedules for the ArduPilot
+fixed wing scripted aerobatics system. If you develop your own
+schedules please contribute to the project by submitting them for
+inclusion in this directory.
+
+## Usage
+
+To use one of these schedules in SITL copy it (or use a symbolic link)
+to a file called trickNN.txt in your scripts/ directory (the same
```

Comment:
The sample scripts say to put copy the script to the root. This seems to be a contradiction.

Reply:
historical from previous commits...not mine...but will take the opportunity to fix

Reply:
need to do this later....the git rework is an issue...will fix in another PR